src/Neo/SmartContract/ApplicationEngine.Helper.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// ApplicationEngine.Helper.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Neo.SmartContract.Native;
+using Neo.VM;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Neo.SmartContract
+{
+    public partial class ApplicationEngine : ExecutionEngine
+    {
+        public string? GetEngineErrorInfo()
```

Comment:
Avoid public for now?

Reply:
> internal?

Works for me, i think we should avoid public method, unless there is a certain usage scenario outside.

Reply:
I am hesitating. I wrote this for public usage.

Reply:
Then keep it this way, all my comment is not blocking one, you do as you wish.